src/sass.js

```diff
@@ -0,0 +1,11 @@
+import { StyleSheet } from 'react-native';
```

Comment:
For the style files, lets call them styles.js - it doesn't make sense to call it a sass file since we're not technically using SASS or SCSS.

Reply:
👍